### ISSUE TITLE
not available in python3 

### ISSUE THREAD

**Issue by icehongssii** [sdushantha/sherlock#24]
I get 
```
  File "sherlock.py", line 17
    print (f"\033[37;1m[\033[91;1m-\033[37;1m]\033[91;1m {errstr}\033[93;1m {err}")
                                                                                 ^
```
I am using 
```
Python 3.5.2
```

**Comment by hoadlck** [comment]
F-Strings are only available in 3.6+. https://www.python.org/dev/peps/pep-0498/

**Comment by sdushantha** [comment]
#28 will let others know that python 3.6+ is required

**Comment by hoadlck** [comment]
OK. So, I think that this issue is closed with #28.  I guess it would be possible to remove all F-Strings and make Sherlock work all of the way back to 2.7, but I am hoping that people will move forward.
